tests/performance/templates/vms.yaml

```diff
@@ -12,6 +12,12 @@ metadata:
 spec:
   runPolicy: AlwaysOn
   enableParavirtualization: true
+  disruptions:
+    {{- if eq (mod . 2) 0 }}
+    restartApprovalMode: Automatic
+    {{- else }}
+    restartApprovalMode: Manual
+    {{- end }}
```

Comment:
Comment this block please. It is not clear why Automatic/Manual should depend on VM number.